packages/tools/kadena-cli/src/prompts/network.ts

```diff
@@ -247,6 +251,34 @@ export const networkSelectOnlyPrompt: IPrompt<string> = async (
   return selectedNetwork;
 };
 
+export const networkSelectWithNonePrompt: IPrompt<string> = async (
+  previousQuestions,
+  args,
+  isOptional,
+): Promise<string> => {
+  const defaultNetwork = previousQuestions.defaultNetwork;
+  const existingNetworks = await getEnsureExistingNetworks();
+
+  const choices: ICustomNetworkChoice[] = existingNetworks.map((network) => ({
+    value: network.value,
+    name: network.name,
+  }));
+
+  if (isNotEmptyString(defaultNetwork)) {
+    choices.unshift({
+      value: 'none',
```

Comment:
if you do this: `const action = network === 'none' ? 'unset' : 'set';`

why not just make the value 'unset' instead of none, since its not visible to the end user anyways?
i would make a choice between internal value usage choice,

Pick none or pick unset:D

Reply:
it makes sense if it's only with prompts but when user wants to pass it directly with args. `--network=unset` looks bit odd.